src/lib.rs

```diff
@@ -451,36 +449,32 @@ impl TimeWindow {
         TimeWindow::new(Time::seconds(a), Time::seconds(b))
     }
 
-    /// Creates time window from start time and duration.
-    /// # Examples
+    /// Creates time window from start time and length.
     ///
+    /// # Examples
     /// ```
-    /// # use tinytime::Duration;
-    /// # use tinytime::Time;
-    /// # use tinytime::TimeWindow;
-    /// let mut x = TimeWindow::from_duration(Time::seconds(1), Duration::seconds(2));
+    /// # use tinytime::*;
+    /// let mut x = TimeWindow::from_length_start(Time::seconds(1), Duration::seconds(2));
     /// assert_eq!(Time::seconds(1), x.start());
     /// assert_eq!(Time::seconds(3), x.end());
     /// ```
     #[must_use]
-    pub fn from_duration(start: Time, duration: Duration) -> Self {
-        TimeWindow::new(start, start.add(duration))
+    pub fn from_length_start(start: Time, length: Duration) -> Self {
```

Comment:
I find this name (and `from_length_end`) a bit unintuitive because its just two words (length, start) without anything connecting them. Maybe `from_length_starting_at` and `from_length_ending_at` could work better?

Reply:
Thanks, nice suggestion